F/FastTransforms/build_tarballs.jl

```diff
@@ -27,6 +27,7 @@ if [[ ${nbits} == 64 ]]; then
 else
     BLAS=openblas
 fi
+make assembly CC=gcc
```

Comment:
Any particular reason for using GCC also on macOS and FreeBSD?

Reply:
Probably just for consistency with the make lib? The lib definitely needs gcc on macOS since it uses OpenMP (I don't think clang links to openmp on macOS).